src/tools/ScenarioMeasurement/Startup/IParser.cs

```diff
@@ -8,6 +8,6 @@ internal interface IParser
     {
         void EnableUserProviders(TraceEventSession user);
         void EnableKernelProvider(TraceEventSession kernel);
-        IList<Counter> Parse(string mergeTraceFile, string processName);
+        IList<Counter> Parse(string mergeTraceFile, string processName, List<int> pids);
```

Comment:
Does this have to be `List<T>` or can it be an `IList<T>`? Or maybe even `IEnumerable<T>`? (The return type here probably doesn't need to be `IList<T>` either. If you wanted to change it to `IEnumerable<T>` that'd be great.)

Reply:
Yes. However the most specific type the parameter can be is IList because it uses the "Contains" method below.